src/app/models/group.ts

```diff
@@ -0,0 +1,6 @@
+import { Member } from "./member";
+
+export class Group{
+  constructor(public members: Member[], lastName: string){}
```

Comment:
is the `lastName` like the groupId?  Maybe just use a `groupName` instead so that we can group members by something more abstract.

Reply:
thanks, that was left from a copy and paste from the member class